app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -145,7 +146,9 @@ class CommonTriageQuestionsController @Inject() (
                   updateIndividualUserType(state, individualUserType)
 
                 val redirectTo =
-                  if (
+                  if (individualUserAmendingRepresenteeFromSelf(state, oldIndividualUserType, individualUserType))
```

Comment:
you don't need something as elaborate as this I don't think. Isn't it just enough to go to this branch if 
```scala
state.fold(_ => false, _.isAmendReturn) && (individualUserType =!= Self)
```
i.e. it's an amend return and they didn't choose `Self`?